composer.json

```diff
@@ -2,40 +2,46 @@
     "name": "olcs/olcs-common",
     "description": "Common library for the OLCS Project",
     "type": "library",
-    "repositories": [
-        {
-            "packagist": false
-        },
-        {
-            "type": "composer",
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "only": [
-                "olcs/olcs-logging",
-                "olcs/olcs-utils"
-            ]
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/OLCS",
-            "type": "composer"
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "type": "composer"
-        }
-    ],
     "require": {
-        "php": ">=7.4",
-        "olcs/olcs-laminas": "~2.5.0 || ~3.0.0"
+        "php": "^7.4",
+        "laminas/laminas-authentication": "^2.6",
+        "laminas/laminas-form": "^2.13",
+        "laminas/laminas-servicemanager": "^2.7",
+        "laminas/laminas-cache": "^2.8",
+        "laminas/laminas-eventmanager": "^2.6",
+        "laminas/laminas-http": "^2.8",
+        "laminas/laminas-i18n": "^2.14",
+        "laminas/laminas-modulemanager": "^2.9",
+        "laminas/laminas-mvc": "^2.7",
+        "laminas/laminas-view": "^2.11",
+        "laminas/laminas-validator": "^2.11",
+        "olcs/olcs-utils": "^5.0",
+        "dvsa/authentication-cognito": "^2.0",
+        "laminas/laminas-console": "^2.7",
+        "laminas/laminas-stdlib": "^2.7",
+        "laminas/laminas-zendframework-bridge": "^1.6",
+        "container-interop/container-interop": "^1.2",
+        "laminas/laminas-psr7bridge": "^0.2.2"
     },
     "require-dev": {
+        "bamarni/composer-bin-plugin": "^1.8",
+        "dms/phpunit-arraysubset-asserts": "^0.5.0",
         "doctrine/annotations": "^1.14.2",
-        "olcs/olcs-devtools": "~4.206.0",
+        "ezyang/htmlpurifier": "^4.17",
+        "johnkary/phpunit-speedtrap": "^4.0",
+        "laminas/laminas-code": "^3.4.1",
+        "laminas/laminas-navigation": "2.13.0",
+        "laminas/laminas-serializer": "^2.5",
```

Comment:
Is this used at all?

Reply:
Laminas\Serializer\AdapterPluginManager has been used in AbstractFormValidationTestCase:63
 $this->sut = $this->getForm()

Reply:
Sorry, I'm missing something, what do you mean?

How is: `Laminas\Serializer\AdapterPluginManager` related to: `$this->sut = $this->getForm()`

Reply:
i mean `Laminas\Serializer` moved to dev dependencies

Reply:
I'm still lost.

Reply:
Looks like `laminas/laminas-mvc` dependent and requires  `Laminas\Serializer`
Tests are failing 
![Screenshot 2023-12-13 at 12 22 31](https://github.com/dvsa/olcs-common/assets/151522199/6a51afd4-11df-49d3-a4a0-c6edc0fbb0f8)

Reply:
`Error: Class 'Laminas\Serializer\AdapterPluginManager' not found

/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-mvc/src/Service/AbstractPluginManagerFactory.php:35
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-mvc/src/Service/AbstractPluginManagerFactory.php:48
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-servicemanager/src/ServiceManager.php:933
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-servicemanager/src/ServiceManager.php:1091
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-servicemanager/src/ServiceManager.php:633
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-servicemanager/src/ServiceManager.php:596
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-servicemanager/src/ServiceManager.php:529
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-modulemanager/src/Listener/ServiceListener.php:206
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-eventmanager/src/EventManager.php:489
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-eventmanager/src/EventManager.php:247
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/vendor/laminas/laminas-modulemanager/src/ModuleManager.php:128
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/test/Common/src/Common/Controller/Lva/AbstractFormValidationTestCase.php:1304
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/test/Common/src/Common/Controller/Lva/AbstractFormValidationTestCase.php:75
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/test/Common/src/Common/Controller/Lva/AbstractFormValidationTestCase.php:140
/Users/sanaullahkhan/Documents/code/olcs/olcs-common/test/Common/src/Common/Controller/Lva/AbstractFormValidationTestCase.php:63
`

Reply:
Okay, so this is a Laminas MVC issue as it's is missing a dependency? Fair enough. Keep it in `devDependencies` but please note when we're doing the v3 upgrade to remove it if it's fixed upstream.